enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -2,15 +2,25 @@
 
 use crate::Layout;
 
-use nolibc::x86_64::error::Number as ErrNo;
-use nolibc::x86_64::syscall::Number as SysCall;
-use nolibc::{ArchPrctlTask, Iovec};
 use sgx_types::ssa::StateSaveArea;
 use span::{Contains, Line, Span};
 
 use core::{mem::size_of, slice::from_raw_parts_mut};
 
 const TRACE: bool = false;
+// The last 4095 numbers are errrnos.
+const ERRNO_BASE: u64 = !0xfff;
+
+// arch_prctl syscalls not available in the libc crate as of version 0.2.69
+enumerate::enumerate! {
+    #[derive(Copy, Clone)]
+    enum ArchPrctlTask: u64 {
+        ArchSetGs = 0x1001,
+        ArchSetFs = 0x1002,
+        ArchGetFs = 0x1003,
+        ArchGetGs = 0x1004,
+    }
+}
```

Comment:
Maybe we should add this to `libc`? @connorkuehl Thoughts?

Reply:
@npmccallum Agreed. I opened #540 for this.